src/stepFunctions/commands/visualizeStateMachine/renderStateMachineGraphCDK.ts

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+
+import { AslVisualizationCDKManager } from './aslVisualizationCDKManager'
+import { ConstructNode } from '../../../cdk/explorer/nodes/constructNode'
+import { getLogger } from '../../../shared/logger'
+import { localize } from '../../../shared/utilities/vsCodeUtils'
+import { showViewLogsMessage } from '../../../shared/utilities/messages'
+import { PreviewStateMachineCDKWizard } from '../../wizards/previewStateMachineCDKWizard'
+import { Window } from '../../../shared/vscode/window'
+
+/**
+ * Renders a state graph of the state machine represented by the given node
+ */
+export async function renderStateMachineGraphCommand(
+    node: ConstructNode | undefined,
+    globalStorage: vscode.Memento,
+    visualizationManager: AslVisualizationCDKManager,
+    executeWizard: boolean,
```

Comment:
Nit: can you cue this off the existence of `node`?

Reply:
Updated to base the wizard execution based on if `node` is undefined.